CIP-dapp-bridge-rework/README.md

```diff
@@ -0,0 +1,1169 @@
+---
+CIP: ????
+Title: Cardano dApp-Wallet Web Bridge
+Status: Proposed
+Category: Wallets
+Authors:
+  - Fell-x27
+Implementors: N/A
+Discussions: N/A
+Created: 2023-11-21
+License: CC-BY-4.0
+---
+
+<!-- TOC -->
```

Comment:
CIP site builders won't be able to parse a Table of Contents here and in fact will create a duplicate.  For this reason it's most portable for CIPs to rely on these site builders to build the table of contents rather than include it in hardcopy... which can get inconsistent with edited headers very easily, especially when other authors submit PRs based on your material.

We have a hardcoded TOC in CIP-0001 (yes it's gotten out of sync several times & I've had to correct it myself a couple times) and you can see how kind of silly it looks here in the generated site: https://cips.cardano.org/cips/cip1/

If you continue to mimic CIP-0030 you can see that the generated TOCs on whatever site builder will contain certain heading levels already; currently:
- [inline: all API headings] https://cips.cardano.org/cips/cip30/
- [sidebar: section headings only] https://developers.cardano.org/docs/governance/cardano-improvement-proposals/CIP-0030

I am hoping @Ryun1 @Crypto2099 @KtorZ will corroborate this but I think the most portable way therefore is to consider the TOC a matter for the presentation layer, not the CIP content itself.

Reply:
That's why it has a <!--TOC--> wrapper, which allows third parties to ignore it if needed. My IDE added it, so I thought it was a standard to resolve issues like you described. It was added to improve navigation in the large document, at least during the 'github only' stage. I also use it in my md-editor. If it's not an option to cut it off by the TOC tag, I think it would be OK if it remains here during the 'github only' stage, and then, if it is included in the CIPs list, I'll remove the TOC to avoid any conflicts with document builders. What do you think?

Reply:
I think that's OK as long as we remember to delete the TOC before any merging, or at least if/when this proposal is in Last Check and after you're sure it's no longer going through your IDE.  still I'll leave this "unresolved" in the meantime so other editors & reviewers can be aware of the issue & help remember this if/when merge time comes.